hydra-node/src/Hydra/HeadLogic/Outcome.hs

```diff
@@ -50,30 +54,55 @@ deriving anyclass instance IsChainState tx => ToJSON (Effect tx)
 instance (ArbitraryIsTx tx, IsChainState tx) => Arbitrary (Effect tx) where
   arbitrary = genericArbitrary
 
+-- | All possible Hydra states.
+data HeadStatus
+  = Idle
+  | Initializing
+  | Open
+  | Closed
+  | FanoutPossible
+  | Final
+  deriving stock (Eq, Show, Generic)
+  deriving anyclass (ToJSON, FromJSON)
+
+instance Arbitrary HeadStatus where
+  arbitrary = genericArbitrary
```

Comment:
This should not need to be here. It's an API specific type and not relevant for the `HeadLogic` at all.